### DISCUSSION TITLE
Do we need snapcraft?

### DISCUSSION THREAD

**Discussion by MatrixCrawler** [warrensbox/terraform-switcher#384] [Ideas]
It seems that the version string in the snapcraft definition that are passed to the build process are outdated AND not matching.
What is this for anyway? @warrensbox 

https://github.com/warrensbox/terraform-switcher/blob/87853c893ef84a8658d9e0794cb3a1232c5dd621/snapcraft.yaml#L2
https://github.com/warrensbox/terraform-switcher/blob/87853c893ef84a8658d9e0794cb3a1232c5dd621/snapcraft.yaml#L45

**Comment by yermulnik** [comment]
Looks like we need to "convert" it to Goreleaser: https://goreleaser.com/customization/snapcraft/

> What is this for anyway?

For `snap` users =) https://snapcraft.io/ — yet another option for Linux users
https://github.com/warrensbox/terraform-switcher/issues?q=is%3Aissue+sort%3Aupdated-desc+snap

**Comment by warrensbox** [comment]
Here is the history behind sanpcraft. Sometime in 2020, someone had requested that I make tfswitch available via snapcraft. 
I created the snapcraft.yml file so that the webhooks on snapcraft.io use the yml to create a release. At first, it appeared that everything was working. But people were complaining that the symlink was not working. I later open a ticket with snapcraft regarding this issue.

The Snapcraft team mentioned that they do not allow symlink creation by tools that are managed by snapcarft.
```
I have a problem creating a symlink from $HOME/bin --> to-the-downloaded-packages (this was my message to their dev team in 2021)
```

Perhaps, now (2024), they allow the creation of symlink by tools installed by snapcraft. 

TLDR: We can **remove** this file for now.
